gateway/project.go

```diff
@@ -71,6 +71,34 @@ func (g *Gateway) CreateProject(ctx context.Context, req *entity.CreateProjectRe
 	return resp.Project, nil
 }
 
+func (g *Gateway) CreateProjectFromTemplate(ctx context.Context, req *entity.CreateProjectFromTemplateRequest) (*entity.CreateProjectFromTemplateResult, error) {
+	gqlReq := gql.NewRequest(`
+		mutation($name: String!, $org: String!, $template: String!, $isPrivate: Boolean, $plugins: [String!], $variables: Json) {
+			createProjectFromTemplate(name: $name, org: $org, template: $template, isPrivate: $isPrivate, plugins: $plugins, variables: $variables) {
+				projectId
+				workflowId
+			}
+		}
+	`)
+
+	g.authorize(ctx, gqlReq.Header)
```

Comment:
I think this bad boi returns an err?
```suggestion
	err := g.authorize(ctx, gqlReq.Header)
	if err != nil {
	    return nil, err
	}
```

Reply:
copy-pasta. Will update the others too